src/styles/base/_override.links.scss

```diff
@@ -1,49 +1,17 @@
-%link-inverse {
-  color: $color-white;
-
-  &:visited {
-    color: $color-white;
-  }
-
-  &:hover,
-  &:active {
-    color: hsla(0, 0%, 100%, 0.77);
-  }
-}
-
-%link,
-%link-darker {
-  color: $color-primary-alt-light;
-
-  &:visited {
-    color: $color-visited;
-  }
-
-  // Override visited with hover
-  &:hover,
-  &:active {
-    color: $color-primary-alt;
-    outline: 0 none;
-  }
-
-  &:focus {
-    box-shadow: 0 0 0 4px $focus-choice-color-highlight,
-      0 0 0 2px $focus-choice-color;
-    border-color: $focus-choice-color;
-    outline: none;
-  }
-
-  &.mct-link--inverse {
-    @extend %link-inverse;
-  }
-}
-
-a,
```

Comment:
This a styling comes from the variable being set to true `$ds-include-base-html-rulesets: true;`